src/coreclr/clrdefinitions.cmake

```diff
@@ -104,7 +106,6 @@ add_definitions(-DFEATURE_DEFAULT_INTERFACES)
 if(FEATURE_EVENT_TRACE)
     add_compile_definitions($<$<NOT:$<BOOL:$<TARGET_PROPERTY:CROSSGEN_COMPONENT>>>:FEATURE_EVENT_TRACE>)
     add_definitions(-DFEATURE_PERFTRACING)
-    add_definitions(-DFEATURE_PERFTRACING_C_LIB)
```

Comment:
This is unused since https://github.com/dotnet/runtime/commit/2ffb39e4779a8b4cfb3d17191f5e959d47ea41ea (this is the only occurrence).